app/helpers/form_helper.rb

```diff
@@ -66,4 +66,9 @@ def us_states_territories
       ['Wyoming', 'WY'],
     ]
   end
+
+  # Filtered list to removve territories that error due to USPS API
```

Comment:
typo in "remove", also maybe we could reword a bit too
```suggestion
  # Filtered list to remove territories that cause errors in the USPS API
```